### DISCUSSION TITLE
Logged in but cant query authenticedUser,  Just retruns null

### DISCUSSION THREAD

**Discussion by thomascanham** [keystonejs/keystone#8919] [Questions] [answered]
If I log in to the Admin UI, everything works as expected.    When I use the GraphQL playground to run some queries and I asked for the authenticated User, It simply returns null.  This is from the playground or the frontend.

My keystone set up is as follows: 
```
`import { config } from '@keystone-6/core';
import { createAuth } from '@keystone-6/auth';
import { statelessSessions } from '@keystone-6/core/session';
import 'dotenv/config';

// Imports for lists here

// Custom values needed for the config
const database = process.env.DATABASE_URL;
const frontend = process.env.FRONTEND_URL;
const sessionSecret = process.env.SESSION_SECRET;
const sessionMaxAge = 60 * 60 * 24 * 1;

const { withAuth } = createAuth({
  listKey: 'User',
  identityField: 'username',
  secretField: 'password',
  sessionData: "name"
});

export default withAuth(
  config({
    db: {
      provider: 'mysql',
      url: `${database}`,
      additionalPrismaDatasourceProperties: {
        relationMode: 'prisma',
      },
    },
    session: statelessSessions({
      maxAge: sessionMaxAge,
      secret: sessionSecret,
    }),
    server: {
      cors: {
        origin: [frontend, "http://localhost:3000"],
        credentials: true,
      }
    },
    ui: {
      isAccessAllowed: (context) => !!context.session?.data,
    },
    lists: {
      //list go her
    },
  })
);
```

Im running the query 

```
query {
  authenticatedItem {
    ... on User {
      id
      username
    }
  }
}
```

Just constantly returning null.   I have actually no idea where to begin on this now.

**Comment by marekryb** [comment, marked as answer]
Hi,

My guess is you are not sending cookies in the playground.
See https://github.com/keystonejs/keystone/discussions/8442#discussioncomment-5484515

**Reply by thomascanham** [reply]
That is exactly what I am looking for, thank you very much!  I wasn't aware that it had changed.

Do you know how to get this working in a Next.js front end?  I am using apollo to to query the data and I can run AuthenticateUserWithPasswor to run correctly and get a success response.   But it is not persisting and I cant query the user on the front end.  I am unsure how to pass the cookies or required tokens across that way.  I have credentials set to 'same-origin'

**Reply by marekryb** [reply]
If you have separate backend (eg. `localhost:4000` / `api.example.com`) and frontend (eg. `localhost:3000` / `example.com`) then, with default cookie settings, they are considered to be different domains and you have to set `credentials: "include"`.

**Reply by thomascanham** [reply]
@marekryb That got it working!   I would like to say a big thank you for taking the time to get back to me on that. It's been puzzling me for a few days now!   Thank you again!
